coordinator/src/test/scala/filodb.coordinator/FilodbClusterSpec.scala

```diff
@@ -29,9 +29,9 @@ class FilodbClusterSpec extends AkkaSpec {
       cluster.clusterActor.isDefined shouldEqual true
       pathElementsExist(clusterActor) should be (true)
     }
+    // Maybe we should just take out this test as we now use Akka's CoordinatedShutdown, not our own shutdown
```

Comment:
Yes, and add a test asserting coordinated shutdown does what it should and I'd add one simulating problems to verify expected behavior/see any issues.